grafana-plugin/src/models/grafana_team/grafana_team.ts

```diff
@@ -29,7 +30,9 @@ export class GrafanaTeamStore extends BaseStore {
 
   @action
   async updateItems(query = '') {
-    const result = await this.getAll();
+    const result = await makeRequest(`${this.path}`, {
+      params: { search: query },
+    });
```

Comment:
why include this change in this PR?

Reply:
search in teams GSelect doesn't work, it's related to current issue, so decided to fix this too